core-bundle/tests/DataContainer/PaletteTest.php

```diff
@@ -33,6 +33,5 @@ public function testPaletteHandling(): void
         $this->assertFalse($palette->hasField('qux'));
 
         $this->assertSame($paletteString, (string) $palette);
-        $this->assertSame($paletteString, $palette->toString());
```

Comment:
Please bring back the `toString()` method and do not remove this line. I have reasoned already why we need this in the other PR.

Reply:
I'm with @leofeyer here (https://github.com/contao/contao/pull/8302#discussion_r2081327357), but that's to be discussed in the other PR (if at all). Restored in 48691806424806f328007bd8721bbb639e04bd03